### ISSUE TITLE
Web preview link on social platform

### ISSUE THREAD

**Issue by mminhlequang** [Eng-Mouaz-M-AlShahmeh/meta_seo#5]
All functions in your package work fine
But just ask for this feature `Web preview link on social platform`
Because maybe bot only read meta tag already set on header index.html
In our logic, meta tags set by package will be later (for create flutter)
Maybe that is problem.

Example Web preview link on social platform:
![image](https://user-images.githubusercontent.com/46816270/217982873-9e0d0826-770c-4aac-86bd-dd0063175c73.png)

Note: same issue with this
https://github.com/Eng-Mouaz-M-AlShahmeh/meta_seo/issues/4

**Comment by Eng-Mouaz-M-AlShahmeh** [comment]
Thanks for the issue,
For the web functionality , browsers read the HTML/XML/XTHML tags if exists and no matter the way you do that, but it is just the tag exist or not, for MetaSEO package just add all meta tags you need into header in HTML and the browser or social platform read this, so you need to be live "not local host". If there are any problem just send screenshots illustrations..

**Comment by mminhlequang** [comment]
Yes i think our package can make more, my issue in case preview link on social, that mean BOT will only read meta info even flutter service not start yet.
Thank you

**Comment by Eng-Mouaz-M-AlShahmeh** [comment]
Welcome,

**Comment by Eng-Mouaz-M-AlShahmeh** [comment]
I update the documentation of the package.
